Client/Frontend/Home/TwoLineCell.swift

```diff
@@ -4,6 +4,111 @@
 
 import UIKit
 
+class ThumbnailCell : UICollectionViewCell {
+    let textLabel = UILabel()
+    let imageView = UIImageView()
+    let margin = 10
+
+    override init() {
+        super.init()
+        setup()
```

Comment:
Remember, `super.init()` already calls `init(frame: CGRect)`. You're doing a double initialization here.